### ISSUE TITLE
ARCore support?

### ISSUE THREAD

**Issue by slayermaster** [alvr-org/PhoneVR#311]
Is there any progress on ARCore tracking? Where should I start from if I were to add ARCore support?

**Comment by AmeliaGL0w0** [comment]
so far arcore is not implemented at all. for the second question, i have no definitive anwser, i was trying to understand the code and research arcore in general to try and add it myself, but thats it so far from me

**Comment by ShootingKing-AM** [comment]
ARCode should be added on Android Project, you can get more quicker dev support on #dev channel in PhoneVR Discord :)

**Comment by ShootingKing-AM** [comment]
Just researched ArCore (Google). It's weirdly supported by Google for only very narrow range of devices (https://developers.google.com/ar/devices). It seems like only flagship models of each OEMs. And phonevr userbase is rarely flagship mobile phones.

Should we really invest time in integrating ArCore ? Not to mention it doesn't even support the device I have myself

**Comment by kotleni** [comment]
@ShootingKing-AM Yeah, google prove a list of supported device. But this list just have tested (?) devices.
So, my two devices not in list but works.

As I see, device just should have Depth API for work.

**Comment by TylerWDoesStuff** [comment]
@ShootingKing-AM there's a github repo for arbitrary supported devices that is scraped from the official list at https://github.com/rolandsmeenk/ARCore-devices
I'm not sure how good ARCore-based 6DoF tracking can be tho since I don't have one that supports Depth API

**Comment by ShootingKing-AM** [comment]
> @ShootingKing-AM there's a github repo for arbitrary supported devices that is scraped from the official list at https://github.com/rolandsmeenk/ARCore-devices I'm not sure how good ARCore-based 6DoF tracking can be tho since I don't have one that supports Depth API

Thanks for the information, i will look into it when am free.

**Comment by DaniXmir** [comment]
hey i know this is kinda old but i did some experimenting with arcore for phone based vr controllers and tracking was surprisingly very good!
vid: https://www.youtube.com/watch?v=TfmfUyrp6fE&t=2s 
also in my testing google's "supported devices page" was very inconsistent, some devices only worked on specific versions of arcore even if they were listed as supporting depth api
